docs/guides/Quick Start/03-orders.md

```diff
@@ -0,0 +1,182 @@
+# Order Overview
+
+The order resource enables merchants to process orders they receive from customers.
+
+You can use the orders resource to do the following:
+
+* Retrieve orders and their line-items
+* Create dummy orders 
```

Comment:
I would not call it "dummy" orders, but illustrate in the corresponding section what types of orders you can create.